docs/extend/plugins.md

```diff
@@ -33,9 +33,6 @@ Follow the instructions in the plugin's documentation.
 
 The sections below provide an inexhaustive overview of available plugins.
 
-<style>
-#content tr td:first-child { white-space: nowrap;}
```

Comment:
Actually, we may need to change this to;

```css
#DocumentationText  tr td:first-child { white-space: nowrap;}
```

This was added to prevent the plugin-names in the first column to wrap, but the ID changed.

Basically to make it look like:

<img width="1161" alt="screen shot 2016-07-14 at 17 36 01" src="https://cloud.githubusercontent.com/assets/1804568/16845525/8ac47174-49e9-11e6-80e8-3aba9e777114.png">

Instead of:

<img width="1147" alt="screen shot 2016-07-14 at 17 36 15" src="https://cloud.githubusercontent.com/assets/1804568/16845538/969f6cb0-49e9-11e6-9e79-994a44e55d32.png">